exercises/practice/simple-cipher/.docs/instructions.md

```diff
@@ -0,0 +1,66 @@
+# Instructions
```

Comment:
This file should probably be rewritten the problem specs to drop the steps and extension.

Reply:
For this exercise, I followed the instructions to move it from the Bash track. The documents and tests taken from Bash. If we're going to rewrite this instruction, it would make sense to do it for both tracks. Let's make a special issue for this.

Reply:
Instructions should be coming from the problem spec, not from bash :)

If the instructions change, it should be in the problem spec and for all tracks.